src/Card/Card.js

```diff
@@ -1,10 +1,15 @@
+import React from 'react';
+
 import Pane from '../Pane';
-import createWithComponent from '../utils/createWithComponent';
 
-const Card = createWithComponent(Pane, 'div', {
-  defaultProps: {
-    borderRadius: 'large'
-  }
-});
+const Card = ({ children, element, ...props }) => (
```

Comment:
Is it necessary to destructure `element` here? Since we're passing in all the props down to `Pane` as it is?

Reply:
Good call, no its not necessary. I think I have just been looking at it for too long.